geotrek/maintenance/tests/test_forms.py

```diff
@@ -60,6 +70,21 @@ def test_end_date_after_start_date(self):
         self.assertFalse(form.is_valid())
         self.assertIn("Begin date is after end date", str(form.errors))
 
+    @skipIf(settings.TREKKING_TOPOLOGY_ENABLED, 'Test without dynamic segmentation only')
```

Comment:
```suggestion
    @skipIf(not settings.TREKKING_TOPOLOGY_ENABLED, 'Test with dynamic segmentation only')
```
It was the other version :sweat_smile: